src/client.ts

```diff
@@ -156,7 +170,7 @@ export class CustomTypesClient {
 	async getAllCustomTypes<TCustomType extends prismicT.CustomTypeModel>(
 		params?: CustomTypesClientMethodParams & FetchParams,
 	): Promise<TCustomType[]> {
-		return await this.fetch<TCustomType[]>("", params);
+		return await this.fetch<TCustomType[]>("customtypes", params);
```

Comment:
Since those are endpoints, I think it makes it more obvious they are by prefixing them with a slash (`/customtypes`), URL class is smart about deduping those in this context.

Reply:
Excluding the `/` prefix is intentional since `/` means an absolute path, where `customtypes` is a relative path.

Not using `/` allows developers to provide an endpoint with a base path, such as:

```
https://example.com/
https://example.com/
https://example.com/
```

Reply:
OK, thinking about it this way, that works for me :)
Thanks for the explaination!

Reply:
I just realized we could use `./customtypes` instead of `customtypes`. It looks more like a URL and is easier to understand. I think it's worth the couple extra characters. 🙂

Reply:
Great idea! When you start reasoning about like file system paths (which is really what a URL is at the end of the day) it all makes more sense.